libscript/src/script-execute.cpp

```diff
@@ -289,6 +320,219 @@ __MCScriptComputeSlotAttributes(const MCResolvedTypeInfo& p_slot_type,
 	r_slot_drop = __MCScriptDropValueRef;
 }
 
+bool
+MCScriptExecuteContext::InvokeForeignVarArgument(MCScriptForeignInvocation& p_invocation,
+                                                 MCScriptInstanceRef p_instance,
+                                                 MCScriptForeignHandlerDefinition *p_handler_def,
+                                                 uindex_t p_arg_index,
+                                                 uindex_t p_arg_reg)
+{
+    // Get the value in the register, this determines the type.
+    MCValueRef t_arg_value = nil;
+    t_arg_value = CheckedFetchRegister(p_arg_reg);
```

Comment:
This should probably just be `MCValueRef t_arg_value = CheckedFetchRegister(p_arg_reg);`